crates/swc/tests/fixture/issues-7xxx/7561/output/1.js

```diff
@@ -0,0 +1,15 @@
+var _class_private_field_get = require("@swc/helpers/_/_class_private_field_get");
+var _class_private_field_init = require("@swc/helpers/_/_class_private_field_init");
+var _x = /*#__PURE__*/ new WeakMap();
+class Foo {
+    test() {
+        var _this_y, _this_y1;
+        _this_y === null || _this_y === void 0 ? void 0 : _class_private_field_get._(_this_y1 = this?.y, _x);
```

Comment:
This is still broken. What's `_this_y` isn't assigned, `this?.y` still remains in the call args.

Reply:
It was a problem with the target ES version.
I added tests to show the result per the target ES version